### PR TITLE
Added Node.js as a code variation

### PR DESCRIPTION
**Pull request by gruselhaus**
Moin (a typical north german greeting),

I think we should add Node.js as a code variation. This describes some challenges in more detail. For example, the Mastodon Bot Challenge is currently categorized under "JavaScript". "Node.js" would be more accurate.

@shiffman What do you think?
@nilsweb FYI

### CODE REVIEW

**Comment by shiffman** [comment]
Yes, I agree! I'm also willing pay for a server and have some sort of process where the node projects are deployed and can be run if that makes sense!

File: _config.yml [hunk 1]
```diff
@@ -40,6 +40,8 @@ code_variations:
     folder: "Processing"
   - name: "JavaScript"
     folder: "JavaScript"
+  - name: "NODE.JS"
```
Comment (shiffman) [inline _config.yml]: Is there a reason why this is all capital?
```suggestion
  - name: "Node.js"
```

Comment (gruselhaus) [reply _config.yml]: There'll no difference. The style will capitalize it anyway.

**Comment by nilsweb** [comment]
Thank you! :tada:

**Pull request merged**